service/src/test/resources/application.yaml

```diff
@@ -115,14 +115,21 @@ tenant:
   header-name: X-Okapi-Tenant
   force-tenant: false
   default-tenant: diku
+
+  # When this and initialize-default-tenant are true, then if the database already exists, then it will be dropped and re-initialized.
+  recreate-default-tenant: false
   initialize-default-tenant: true
+
   domain-packages:
     - org.folio.rest.workflow.components
     - org.folio.rest.workflow.model
   schema-scripts:
 
 okapi:
   url: ${OKAPI_URL:http://localhost:9130}
+  camunda:
```

Comment:
Could you help explain again why both of these paths are required for mod-workflow to communicate with mod-camunda through Okapi? Is there some limitation with Camunda Dashboard or its configuration for the Camunda API?